src/ucp/tag/tag_send.c

```diff
@@ -78,7 +78,7 @@ static ucs_status_t ucp_tag_req_start(ucp_request_t *req, size_t count,
                (length >= rndv_rma_thresh)) ||
                (length >= rndv_am_thresh)) && !is_iov) {
         /* RMA/AM rendezvous */
-        ucp_tag_send_start_rndv(req);
+        req->send.uct.func = proto->start_rndv;
```

Comment:
can also avoid indirect call here - disable rndv by enlarging threshold to maximum in ucp_ep_config_init, in case err handling is required